examples/api/lib/widgets/hardware_keyboard/key_event_manager.0.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+// This example app demonstrates a use case of patching
```

Comment:
I would include a description for at least one hypothetical use-case where a developer would want to patch key handling. If you can think of more then one, then I'd mention multiple use-cases. Examples help a lot with gaining context for an example that a developer is about to read.

Reply:
I really can't think of any, because if there's a usage that's is easy to explain and not possible with the current keyboard API, I would think about new API to support it. Even the usage you mentioned, "show popup indications for what the user presses", should be possible with `HardwareKeyboard.instance.addHandler`. I will add more documentation to the other methods though. Thanks for your feedback.